README.md

```diff
@@ -1,6 +1,6 @@
 # q2galaxy
 
-[![](https://github.com/qiime2/q2galaxy/workflows/ci/badge.svg)](https://github.com/qiime2/q2galaxy/actions/workflows/ci.yml)
+[![](https://github.com/qiime2/q2galaxy/workflows/ci-dev/badge.svg)](https://github.com/qiime2/q2galaxy/actions/workflows/ci-dev.yml)
```

Comment:
```suggestion
![](https://github.com/qiime2/q2galaxy/workflows/ci-dev/badge.svg)
```
I'm not sure why there were two of these SVG badges (not your fault at all, it was there prior to this pull request), but let's just remove this extraneous one.